### ISSUE TITLE
Test suite fails if bzip2 is not installed

### ISSUE THREAD

**Issue by eserte** [glasswalk3r/archive-tar-wrapper-perl#11]
On a very minimal debian/jessie system running in a docker container:
```
2018/07/09 18:35:52 /bin/tar jx -f /root/.cpan/build/Archive-Tar-Wrapper-0.31-NMaTJ4/t/data/foo.tar.bz2 failed: tar (child): bzip2: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
/bin/tar: Child returned status 2
/bin/tar: Error is not recoverable: exiting now

#   Failed test 'opening compressed tarfile'
#   at t/007bzip.t line 17.

#   Failed test 'find 001Basic.t'
#   at t/007bzip.t line 19.

#   Failed test 'find ./001Basic.t'
#   at t/007bzip.t line 20.
# Looks like you failed 3 tests of 5.
t/007bzip.t ...... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/5 subtests 
```
The Dockerfile looked like this:
```
# docker build -t perl-test . && docker run perl-test
FROM debian:jessie
RUN apt-get -y update
RUN apt-get -y install perl-modules
RUN apt-get -y install make
CMD cpan -t ARFREITAS/Archive-Tar-Wrapper-0.31.tar.gz
```

**Comment by eserte** [comment]
I think skipping is best --- at least the module still works in other situations (.tar.gz and plain .tar).

**Comment by glasswalk3r** [comment]
Thanks! I agree is the mos portable option!
Fixed with commit b0973ecc3d1d0d9b41a107a293ade5d0242ba9ce.
